cdc/sink/mq.go

```diff
@@ -75,11 +76,15 @@ type mqSink struct {
 	resolvedReceiver     *notify.Receiver
 
 	statistics *Statistics
+
+	role util.Role
+	id   model.ChangeFeedID
 }
 
 func newMqSink(
 	ctx context.Context, credential *security.Credential, mqProducer producer.Producer,
-	filter *filter.Filter, replicaConfig *config.ReplicaConfig, opts map[string]string, errCh chan error,
+	filter *filter.Filter, replicaConfig *config.ReplicaConfig, opts map[string]string,
+	errCh chan error, role util.Role,
```

Comment:
I think this role could also be put into context, like changefeed. Also it's weird to put it in util, maybe model/owner is a good place for it? But util looks weird.

Reply:
Yes, we can put it into context, but `model/owner` is not a good place, this does not belong to `owner`.

Reply:
Hmmmmm Then we may need to define clearly where it is. Because it doesn't look like a util.

Reply:
put it into `model/owner` would introduce cyclic import.

Take a look at `ChangefeedIDFromCtx`, feel so weird, it returns `string` assigned to `model.ChangefeedID`. I think this is to resolve the cyclic import.

But for role, this solution does not handle the problem.

Reply:
I would prefer refact the `model`  related definitions in another PR.